### PR TITLE
(maint) Use shared helpers in Azure

### PR DESCRIPTION
**Pull request by Iristyle**

### CODE REVIEW

**Comment by Iristyle** [comment]
We still have to move over to the standard `Build-Container` helper to complete this work, but saving that for a separate PR since it will be a little bit of work.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -8,108 +8,100 @@ pr:
...
-  displayName: Test Puppet Agent Ubuntu
-  name: test_puppet_agent_ubuntu
-- task: PublishTestResults@2
-  displayName: Publish Puppet Agent Ubuntu test results
-  inputs:
-    testResultsFormat: 'JUnit'
-    testResultsFiles: 'docker/TEST-*.xml'
-    #mergeTestResults: false # Optional
-    testRunTitle: Puppet Agent Ubuntu Test Results
+    Build-Container -Namespace $ENV:NAMESPACE -Base 'ubuntu'
```
Comment (nwolfe) [inline azure-pipelines.yml]: I don't recognize this `-Base` option ?

Comment (nwolfe) [reply azure-pipelines.yml]: Oh it's not coming from the shared one, it's coming from the local one

Comment (Iristyle) [reply azure-pipelines.yml]: That's a leftover from the existing `Build-Container` helper that I left alone for now (rather than from the shared one)

**Pull request merged**